package.json

```diff
@@ -65,7 +65,7 @@
     "@ndhoule/extend": "^2.0.0",
     "@pyroscope/nodejs": "^0.2.9",
     "@rudderstack/integrations-lib": "^0.2.13",
```

Comment:
Do we need to update integrations-lib version ?

Reply:
No need KC, the util I imported was not from integrations-lib. It is imported from transformer utils